README.md

```diff
@@ -8,7 +8,7 @@
 
 t8code (spoken as "tetcode") is a C/C++ library to manage parallel adaptive meshes with various element types.
 t8code uses a collection (a forest) of multiple connected adaptive space-trees in parallel and scales to at least one million MPI ranks and over 1 Trillion mesh elements.
-It is licensed under the GNU General Public License 2.0 or later. Copyright (c) 2015 the developers.
+It is licensed under the GNU General Public License 2.0 or later. Copyright (c) 2023 the developers.
```

Comment:
Please do not change the copyright message - especially if the PR is not related to Copyright.

You are right that we should update the year. Following https://www.gnu.org/prep/maintain/html_node/Copyright-Notices.html we should list all years where we did changes (so all from 2015 to 2023). But let's inform ourselves here on the best practices and then do a stand-alone PR.

Reply:
Fair enough. I'll revert this change.